### PR TITLE
Migration of panther_analysis_tool to dedicated repo

### PR DESCRIPTION
**Pull request by nhakmiller**
Moving the `panther_analysis_tool` from the `panther-analysis` repo to here so it can have a dedicated space.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,285 @@
...
+  <i>Command Line Panther Analysis Management</i>
+</p>
+
+<p align="center">
+  <a href="https://docs.runpanther.io">Documentation</a> |
+  <a href="https://docs.runpanther.io/quick-start">Quick Start</a>
+</p>
+
+<p align="center">
+  <a href="https://gitter.im/runpanther/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge"><img src="https://badges.gitter.im/runpanther/community.svg" alt="Gitter"/></a>
```
Comment (austinbyers) [inline README.md]: Remove Gitter link, replace with Slack

File: README.md [hunk 2]
```diff
@@ -1,2 +1,285 @@
...
+  <a href="https://circleci.com/gh/panther-labs/panther-analysis"><img src="https://circleci.com/gh/panther-labs/panther-analysis.svg?style=svg" alt="CircleCI"/></a>
```
Comment (austinbyers) [inline README.md]: Change to point to `panther_analysis_tool` (both the link and the img src)

File: README.md [hunk 3]
```diff
@@ -1,2 +1,285 @@
...
+  <a href="https://cla-assistant.io/panther-labs/panther-analysis" alt="CLA Assistant"><img src="https://cla-assistant.io/readme/badge/panther-labs/panther-analysis"/></a>
```
Comment (austinbyers) [inline README.md]: I'm not sure if we have CLA enabled for this repo yet, but if so change the link

File: buildspec.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+version: 0.2
```
Comment (austinbyers) [inline buildspec.yml]: We already have a `.circlei/config.yml` file, I don't think we need this one?

File: config.yml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+version: 2
```
Comment (austinbyers) [inline config.yml]: Again, don't think we need this file

File: README.md (hunk 3 above)
Comment (nhakmiller) [reply README.md]: I don't think so either, removing it for now.

**Pull request merged**